Model/Ui/ConfigProvider.php

```diff
@@ -87,8 +97,31 @@ public function getConfig()
             "environment" => $this->_config->getEnvironment(),
             "title" => $this->_config->getTitle(),
             "inContextCheckoutEnabled" => (bool)$this->_config->isInContextCheckout(),
-            "iframe" => $this->_config->isInContextCheckout()
+            "iframe" => $this->_config->isInContextCheckout(),
+            "isTokenisationEnabled" => $this->_canCustomerSeeTokenisationOption(),
+            "isCustomerWantTokenisation" => $this->_isCustomerSelectedTokenisationBefore()
         ];
         return $config;
     }
+
+    /**
+     * check database customer before selected tokenisation
+     */
+    protected function _isCustomerSelectedTokenisationBefore()
+    {
+        return false;
+    }
+
+    protected function _canCustomerSeeTokenisationOption()
+    {
+        return $this->_config->isTokenisationEnabled() && $this->_isCustomerLogin();
+    }
+
+    protected function _isCustomerLogin()
```

Comment:
Just for readability, I think you mean this function to be _isCustomerLog**ged**In ?